src/Pmi/Controller/DefaultController.php

```diff
@@ -25,36 +24,10 @@ class DefaultController extends AbstractController
         ['clientTimeout', '/client-timeout', [ 'method' => 'GET' ]],
         ['agreeUsage', '/agree', ['method' => 'POST']],
         ['groups', '/groups'],
-        ['selectSite', '/site/select', ['method' => 'GET|POST']],
         ['hideTZWarning', '/hide-tz-warning', ['method' => 'POST']],
-        ['patientStatus', '/participant/{participantId}/patient/status/{patientStatusId}', ['method' => 'GET']],
-        ['mockLogin', '/mock-login', ['method' => 'GET|POST']]
+        ['patientStatus', '/participant/{participantId}/patient/status/{patientStatusId}', ['method' => 'GET']]
     ];
 
-    public function homeAction(Application $app)
-    {
-        $checkTimeZone = $app->hasRole('ROLE_USER') || $app->hasRole('ROLE_ADMIN') || $app->hasRole('ROLE_AWARDEE') || $app->hasRole('ROLE_DV_ADMIN') || $app->hasRole('ROLE_BIOBANK') || $app->hasRole('ROLE_SCRIPPS') || $app->hasRole('ROLE_AWARDEE_SCRIPPS') ;
-        if ($checkTimeZone && !$app->getUserTimezone(false)) {
-            $app->addFlashNotice('Please select your current time zone');
-            return $app->redirectToRoute('settings');
-        }
-        if ($app->hasRole('ROLE_USER') || ($app->hasRole('ROLE_AWARDEE') && $app->hasRole('ROLE_DV_ADMIN'))) {
-            return $app['twig']->render('index.html.twig');
-        } elseif ($app->hasRole('ROLE_AWARDEE')) {
-            return $app->redirectToRoute('workqueue_index');
-        } elseif ($app->hasRole('ROLE_DV_ADMIN')) {
-            return $app->redirectToRoute('problem_reports');
-        } elseif ($app->hasRole('ROLE_ADMIN')) {
-            return $app->redirectToRoute('admin_home');
-        } elseif ($app->hasRole('ROLE_DASHBOARD')) {
-            return $app->redirectToRoute('dashboard_home');
-        } elseif ($app->hasRole('ROLE_BIOBANK') || $app->hasRole('ROLE_SCRIPPS')) {
-            return $app->redirectToRoute('biobank_home');
-        } else {
-            return $app->abort(403);
-        }
-    }
-
     public function dashSplashAction(Application $app)
```

Comment:
Should `dashSplashAction`, `logoutAction`, and `loginReturnAction` be removed/redirected now?

Reply:
I'll check and see if we can remove those without affecting the silex test cases.

Reply:
We don't have the `loginReturnAction` in Symfony side so went ahead removed and redirected silex logout, dashSplash, timeout, and clientTimeout routes.